README.md

```diff
@@ -256,6 +261,13 @@ reqData = new HashMap<String,String>();
 reqData.put("limit", "10");
 
 DataApi dataApi = new DataApi("https://data.learnosity.com/latest/itembank/items", sec, consumerSecret, reqData, "get");
+
+// you can also configure the Data API with a requestConfig as above
+RequestConfig requestConfig = RequestConfig.custom().setSocketTimeout(1000).build();
+
+dataApi.modifyRequest(requestConfig);
```

Comment:
Think based on this: https://github.com/Learnosity/learnosity-sdk-java/pull/3/files#diff-8b8d082620907b03e81bc9416495c430R144

the method should be `modifyRemote` ?